libraryapi/pergamum.py

```diff
@@ -42,16 +43,17 @@ def __init__(self, base_url: str) -> None:
                 f"{base_url}/web_service/servidor_ws.php?wsdl",
                 transport=Transport(session=session),
             )
-        except HTTPError as e:
+        except HTTPError as error:
             raise PergamumWebServiceException(
```

Comment:
Esse rename não ganhou muita semantica porque é uma variável de contexto , o comum seria o "e" mesmo nesse caso, mas ok

Reply:
Mudar de "e" para "error", foi só para o pylint não ficar avisando "Variable name "e" doesn't conform to snake_case naming style".